src/views/CountryRiskWatch/RiskBarChart/WildfireChart/index.tsx

```diff
@@ -220,6 +220,15 @@ function WildfireChart(props: Props) {
             ref={chartContainerRef}
         >
             <svg className={styles.svg}>
+                <text
+                    className={styles.yAxisLabel}
+                    textAnchor="middle"
+                    transform={`translate(${(chartMargin.left - CHART_OFFSET) / 3},
+                            ${(chartBounds.height - chartMargin.bottom - chartMargin.top) / 2})
+                            rotate(-90)`}
+                >
+                    {strings.monthlySeverityRating}
+                </text>
```

Comment:
We should either create a separate component for this or implement it in the `ChartAxes` component